### ISSUE TITLE
Missing ServiceType definition

### ISSUE THREAD

**Issue by m-elseberg** [open-pioneer/trails-core-packages#79]
### Bug description

The documentation on [How to Use a Service](https://github.com/open-pioneer/trails-starter/blob/main/docs/tutorials/HowToUseAService.md#using-a-service-from-another-service) suggests using the `ServiceType` type when referencing a service from another service:

```ts
import { ServiceOptions, ServiceType } from "@open-pioneer/runtime";
```

However, no such type exists ([index.ts](https://github.com/open-pioneer/trails-core-packages/blob/main/src/packages/runtime/index.ts)):

```
Module '"@open-pioneer/runtime"' has no exported member 'ServiceType'.ts(2305)
```

### Validations

- [X] Read and follow our [code of conduct](https://github.com/open-pioneer/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [contributing guidelines](https://github.com/open-pioneer/trails-starter/blob/main/docs/Contributing.md)
- [X] Check that there isn't already a similar issue

**Comment by mbeckem** [comment]
Right, this is outdated. We'll fix it soon, for you should just import the needed type directly.
